### DISCUSSION TITLE
Welcome to Pelican's Discussions!

### DISCUSSION THREAD

**Discussion by lancepioch** [pelican-dev/panel#1] [Announcements]
## 👋 Welcome Pelican Community!
  We’re using GitHub Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by PascaleBeier** [comment]
Hey @lancepioch, I would love to support you with my dev time. We are running a ptero fork on palhost.de and are looking to migrate in the long run. I'll just tag along for now, let's see how that goes. :)
